modules/ipi-install-network-requirements.adoc

```diff
@@ -5,7 +5,7 @@
 [id='network-requirements_{context}']
 = Network requirements
 
-Installer-provisioned installation of {product-title} involves several network requirements by default. First, installer-provisioned installation involves a non-routable `provisioning` network for provisioning the operating system on each bare metal node and a routable `baremetal` network. Since installer-provisioned installation deploys `ironic-dnsmasq`, the networks should have no other DHCP servers running on the same broadcast domain. Network administrators must reserve IP addresses for each node in the {product-title} cluster.
+Installer-provisioned installation of {product-title} involves several network requirements by default. First, installer-provisioned installation involves a non-routable `provisioning` network for provisioning the operating system on each bare metal node and a routable `baremetal` network. By default, installer-provisioned installation deploys `ironic-dnsmasq` with DHCP enabled for the `provisioning` network. No other DHCP servers should be running on the `provisioning` network. Network administrators must reserve IP addresses for each node in the {product-title} cluster for the `baremetal` network on an external DHCP server.
```

Comment:
This is a section about DHCP-related requirements and explanations. Should it better be titled _as Dynamic Host Configuration Protocol (DHCP)_? Which would be consistent with then having the NTP-related reqs/explanations section. What do you think?

Reply:
We could also cross reference to https://docs.openshift.com/container-platform/4.7/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#additional-install-config-parameters_ipi-install-configuration-files which shows that you actually *can* have a DHCP server on the provisioning network, but only if you set `provisioningNetwork: unmanaged`

However that scenario isn't documented other than the install-config option, and I noticed there's an issue with those docs (should be Managed/Unmanaged, not managed/unmanaged ref https://github.com/openshift/installer/blob/master/pkg/asset/installconfig/baremetal/baremetal.go#L24)

So perhaps it's best to leave this as-is until such time as we add more details around that (AFAIK it's not a common requirement)

Reply:
I've reordered the content in the network requirements section, and also broke step 11 into sub steps per feedback from Dan Macpherson. Please have a look.

Reply:
@mburke5678, could you have another look at this?

Reply:
@johnwilkins it looks great and thanks for addressing the nmcli commands for the networks, this is so much better and accurate since if you didn't use the provisioning network you had to figure out yourself what to run from the nmcli commands.